pkg/controller/endpoint/endpoints_controller.go

```diff
@@ -114,6 +114,10 @@ type EndpointController struct {
 	serviceStore cache.StoreToServiceLister
 	podStore     cache.StoreToPodLister
 
+	// personalPodInformer is used to hold a personal informer.  If we're using a normal shared informer, then the informer will be
+	// started for us.  If we have a personal informer, we must start it ourselves
```

Comment:
Can you please add explicit comment that if you start the controller using "NewEndpointController" (i.e. passing SharedInformer), this will be null?

Also nit: - can you split lines so that they are not so long?